lms/djangoapps/onboarding/models.py

```diff
@@ -253,6 +253,14 @@ def admin_info(self):
         """
         return "%s" % self.admin.email if self.admin else "Administrator not assigned yet."
 
+    @property
+    def first_learner(self):
+        """
+        :return: organization's first learner
+        """
+        org_first_learner = self.extended_profile.filter(is_first_learner=True).first()
```

Comment:
Does this model have `extended_profile` attribute/object? This is model does not inherit from custom model or class neither it has any `extended_profile` field/property/object.

Can you please verify, it works.

Reply:
Just saw the comment https://github.com/philanthropy-u/edx-platform/pull/1602#discussion_r440702060. Marking it resolved.

Reply:
yes it has great thanks